src/connectors/zendesk/functions/update_ticket.ts

```diff
@@ -75,44 +67,23 @@ export default DefineConnector({
       },
       zendesk_access_token: {
         type: Schema.slack.types.oauth2,
-        description: "Zendesk Credential to use",
         title: "Zendesk Access Token",
       },
     },
     required: ["ticket_id", "zendesk_access_token"],
   },
   output_parameters: {
     properties: {
-      requester_id: {
-        type: Schema.types.integer,
-        description: "The ticket's requester ID",
-        title: "Requester ID",
-      },
-      requester_email: {
-        type: Schema.types.string,
-        description: "The ticket's requester email",
-        title: "Requester email",
-      },
-      ticket_id: {
-        type: Schema.types.integer,
-        description: "The ticket ID",
-        title: "Ticket ID",
-      },
-      ticket_url: {
-        type: Schema.types.string,
-        description: "The ticket URL",
-        title: "Ticket URL",
-      },
+      requester_id: { type: Schema.types.integer, title: "Requester ID" },
+      requester_email: { type: Schema.types.string, title: "Requester email" },
+      ticket_id: { type: Schema.types.integer, title: "Ticket ID" },
+      ticket_url: { type: Schema.types.string, title: "Ticket URL" },
```

Comment:
The descriptions seem to have gotten removed, this should not be a breaking change since it does not affect using the connector